### DISCUSSION TITLE
Calculating distance between turbines

### DISCUSSION THREAD

**Discussion by labbatessa** [NatLabRockies/floris#949] [Q&A] [answered]
Hello, I used the gch.yaml file in the floris/examples/inputs directory to walk through the "Introductory Concepts" section of "User Reference" contained within the FLORIS documentation, and to supplement the code I've written, I'm interested in calculating the distances between turbines in GPS space. However, I'm finding that I need a GPS reference point with latitude and longitude for the origin of the original Cartesian coordinates (0,0). With that being said, I was wondering if there are real-world latitude longitude coordinates of the Cartesian origin (0,0), or if the coordinates for the turbine locations in the x- and y-directions in the gch.yaml file are completely arbitrary. Thank you!

**Comment by misi9170** [comment, marked as answer]
Hi @labbatessa ,

The x,y coordinates of wind farms defined in FLORIS are arbitrary---you can take them to be at any (latitude, longitude) location. The dimensions of x and y are in meters. To convert between the Euclidean coordinates used in FLORIS and lat/lon (if you have a specific location you would like to model), you could look into a package like [Geopandas](https://geopandas.org/en/stable/).

**Reply by labbatessa** [reply]
Sounds good, thank you @misi9170!
